File: archive/j/julia/fizz-buzz.jl
#!/usr/bin/julia

for i = 1:100
    str = i % 3 == 0 ? "Fizz" : ""
    str *= i % 5 == 0 ? "Buzz" : ""
    if isempty(str)
        str = i
    end
    println(str)
end

File: archive/w/wu/fizz-buzz.lua
return (function()
  local i = 1
  
  while (i <= 100) do
    repeat
    local o = ""
    
    if ((i % 3) == 0) then
      o = (o .. "Fizz")
    end
    
    if ((i % 5) == 0) then
      o = (o .. "Buzz")
    end
    
    if (((i % 3) ~= 0) and ((i % 5) ~= 0)) then
      o = (o .. i)
    end
    
    print(o)
    i = (i + 1)
    until true
  end
  
  return {
    i = i,
  }
end)()